sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceStrategy.scala

```diff
@@ -635,3 +641,16 @@ object DataSourceStrategy {
     }
   }
 }
+
+/**
+ * Find the column name of an expression that can be pushed down.
+ */
+private[sql] object PushableColumn {
```

Comment:
@dbtsai, if this is `private[sql]` for the test purpose, we could just `private[datasources]`.

Reply:
execution package is private. We can remove `private[sql]` completely.

Reply:
Okay. Addressed. Thanks.